_posts/2018-10-16-film-club-9-the-darjeeling-limited.md

```diff
@@ -0,0 +1,67 @@
+---
```

Comment:
Here is a bit of what I wrote up over the weekend for this week's homework, @kousun12 

All the rest of the PR is just some formatting cleanup.